src/utils/AttestedClaimUtils/AttestedClaimUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { IAttestedClaim } from '@kiltprotocol/sdk-js'
+import * as Claims from '../../state/ducks/Claims'
+
+// eslint-disable-next-line import/prefer-default-export
+export const buildMatchingAttestedClaims = (
```

Comment:
Is this function still necessary?
We now store the full AttestedClaim when importing Attestations.